components/modal/config/DialogOptions.cs

```diff
@@ -128,25 +128,15 @@ internal string GetWidth()
             }
         }
 
-        internal string GetWrapClassNameExtended(Modal modal = null)
+        internal string GetWrapClassNameExtended(ModalStatus status)
         {
             var classNameArray = new List<string>();
-            if (modal == null)
-            {
-                classNameArray.AddIf(
-                        !string.IsNullOrWhiteSpace(WrapClassName),
-                        WrapClassName)
-                    .AddIf(Centered, $"{PrefixCls}-centered")
-                    .AddIf(Rtl, $"{PrefixCls}-wrap-rtl");
-
-                return string.Join(' ', classNameArray);
-            }
 
             classNameArray.AddIf(
-                    !string.IsNullOrWhiteSpace(modal.WrapClassName),
```

Comment:
`GetWrapClassNameExtended` is only used in Dialog, and Modal is always null, so this code can clean up.